templates/templates/blockified/cart.html

```diff
@@ -0,0 +1,8 @@
+<!-- wp:template-part {"slug":"header"} /-->
+<!-- wp:group {"layout":{"inherit":true}} -->
+<div class="wp-block-group">
+	<!-- wp:paragraph -->
+	<p>Cart blockified</p>
```

Comment:
What are these paragraphics for @wavvves ?

Reply:
This was left over code from the POCs, but I think these are unnecessary.

@gigitux can these files within the blockified folder be removed? What is the purpose for them? 🤔